dspace-spring-rest/src/test/java/org/dspace/app/rest/DiscoveryRestControllerIT.java

```diff
@@ -1660,7 +1660,7 @@ public void discoverSearchObjectsTestForEmbargoedItemsAndPrivateItems() throws E
                                       .withIssueDate("1990-02-13")
                                       .withAuthor("Smith, Maria").withAuthor("Doe, Jane").withAuthor("Testing, Works")
                                       .withSubject("TestingForMore").withSubject("ExtraEntry")
-                                      .makePrivate()
+                                      .makeUnDiscoverable()
```

Comment:
Consider renaming the method from "private" to "undiscoverable" to match the method names in use.